microsoft-edge/extensions-chromium/developer-guide/sidebar.md

```diff
@@ -38,7 +38,9 @@ Extensions can optionally use the sidebar API to show a custom UI in the Microso
 
 Features of the Sidebar API include:
 
-* The sidebar remains open while navigating between tabs.
+* The sidebar remains open while navigating between tabs.<!-- todo: fix/reconcile -->
```

Comment:
is there an Issue to link to?

went live July 7, 2023:
[Sidebar API for extensions is now live! · microsoft/MicrosoftEdge-Extensions · Discussion #95](https://github.com/microsoft/MicrosoftEdge-Extensions/discussions/95) 
 
Search for Issues w/ "tab":
https://github.com/microsoft/MicrosoftEdge-Extensions/discussions?discussions_q=is%3Aopen+tab
possible contact in apparently related Issue: 
[Cannot remove/update NTP tab issue · microsoft/MicrosoftEdge-Extensions · Discussion #217](https://github.com/microsoft/MicrosoftEdge-Extensions/discussions/217)

https://github.com/ManikanthMSFT
@ManikanthMSFT

Reply:
I think this is the issue we should link to: https://github.com/microsoft/MicrosoftEdge-Extensions/issues/142 from your note below.